src/app/auth/styles.tsx

```diff
@@ -0,0 +1,23 @@
+import styled from 'styled-components';
+import COLORS from '@/styles/colors';
+
+export const FormDiv = styled.div`
```

Comment:
Clarifying question: Is this pulled from Alvaro's interest form PR?

Reply:
I'm assuming you're referring to https://github.com/calblueprint/immigration-justice-project/pull/25. If so, no, I created my own styled div and didn't intentionally pull this from the PR (am i supposed to?). sorry if the component name is a bit confusing; let me know if there's a better naming convention to follow, or I'll try to think of a better name.

Reply:
Oh no worries! I was just curious because of the name and the fact that there's a similar div in his PR (I believe he calls it `CaseInterestContainer`.) This is totally fine though, we can potentially consider condensing them into one component once both PRs are merged in.